metrics-proxy/src/main/java/ai/vespa/metricsproxy/metric/dimensions/BlocklistDimensions.java

```diff
@@ -0,0 +1,73 @@
+package ai.vespa.metricsproxy.metric.dimensions;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author olaa
+ */
+public class BlocklistDimensions {
+
+    private BlocklistDimensions() {}
+
+    /**
+     * Deployment related metrics - most of which are redundant
+     * E.g. app/applicationName/tenantName/instanceName is already included in applicationId
+     */
+    public static final String APP = "app";
```

Comment:
You could avoid "public static final" for each constant if you use an enum with a constructor that takes a String. I think you could also avoid repeating all the constants by using EnumSet.all().